src/providers/postgres/qgspostgresdataitemguiprovider.cpp

```diff
@@ -190,6 +190,30 @@ QWidget *QgsPostgresDataItemGuiProvider::createParamWidget( QgsDataItem *root, Q
   }
 }
 
+QString QgsPostgresDataItemGuiProvider::typeNameFromLayer( const QgsPostgresLayerProperty &layer )
+{
+  switch ( layer.relKind )
+  {
+    case Qgis::PostgresRelKind::View:
+      return tr( "View" );
+
+    case Qgis::PostgresRelKind::MaterializedView:
+      return tr( "Materialized View" );
+
+    case Qgis::PostgresRelKind::NotSet:
+    case Qgis::PostgresRelKind::Unknown:
+    case Qgis::PostgresRelKind::OrdinaryTable:
+    case Qgis::PostgresRelKind::Index:
```

Comment:
Is it correct to return "Table" for index, sequence etc?

Reply:
It's not :laughing: -- but it's the same behavior as before this PR and I didn't want to go too deep with altering existing logic.